src/mpfao/3D/tdympfao3D_core.c

```diff
@@ -979,3 +1032,317 @@ PetscErrorCode TDyComputeTransmissibilityMatrix3DMesh(TDy tdy) {
   PetscFunctionReturn(0);
 
 }
+
+/* -------------------------------------------------------------------------- */
+/// Computes unit vector joining upwind and downwind cells that share a face.
```

Comment:
Maybe we should mention that this unit vector points from the centroid of the upwind cell to that of the downwind cell.